src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/ErrorConsumerImplTest.java

```diff
@@ -26,6 +30,9 @@
     @Mock
     private ApplicationLogger logger;
 
+    @Mock
+    private SlackMessagingService slackMessagingService;
```

Comment:
I think most of the changes in here can be reverted as slackMessagingService is no longer used by ErrorConsumerImpl

Reply:
Will keep this to verify that it is not being called in error mode.

Reply:
I think we should be ok removing it as it's not used by the class. it might cause confusion for someone who looks at the code after us and assume that the class is using the service, @ICoxCH what do you think?

Reply:
I agree with Mark, SlackMessagingService isn't even present in the class so asserting this isn't called doesn't add anything.
I'm surprised Mockito doesn't complain if i'm honest